README.md

```diff
@@ -42,20 +42,24 @@ Versions can be restricted from updating within the `pyproject.toml` using stand
 
 To get started:
 
-1. Create and activate a [virtual environment](https://docs.python.org/3/library/venv.html):
+1. Create and activate a [virtual environment](https://docs.python.org/3/library/venv.html).
+   
+**Note:** It is important that this environment is created with `Python 3.11`. It cannot be any higher or lower Python version due to `coldfront` limitations at the time of writing. You may note we ensure this requirenment in the `pyproject.toml` file by `requires-python = "~=3.11"`, but wish to mention it again explicitly here. A tool like `pyenv` that has any Python version activated tht is different from `Python 3.11`, can bypass this requirenment and lead to side effects.
```

Comment:
Not entirely sure what the last sentence is meant to mean, but it is probably not necessary in any case
```suggestion
**Note:** It is important that this environment is created with `Python 3.11`. It cannot be any higher or lower Python version due to `coldfront` limitations at the time of writing. You may note we ensure this requirement in the `pyproject.toml` file by `requires-python = "~=3.11"`, but wish to mention it again explicitly here.
```